sql/moz-fx-data-shared-prod/firefox_ios_derived/retention_v1/schema.yaml

```diff
@@ -0,0 +1,81 @@
+fields:
+
+- mode: NULLABLE
+  name: metric_date
+  type: DATE
+  description: This is 28 days before processing.
+
+- mode: NULLABLE
+  name: first_seen_date
+  type: DATE
+  description: When the client has been seen for the first time.
+
+- name: app_name
+  type: STRING
+  mode: NULLABLE
+  description: App name the client is using.
+
+- mode: NULLABLE
+  name: channel
+  type: STRING
+  description: Release channel of the app the client is using.
+
+- name: country
+  type: STRING
+  mode: NULLABLE
+  description: Client's country on the metric date.
+
+- name: adjust_ad_group
+  type: STRING
+  mode: NULLABLE
+  description: Adjust Ad Group the client is attributed to.
+
+- name: adjust_campaign
+  type: STRING
+  mode: NULLABLE
+  description: Adjust Campaign the client is attributed to.
+
+- name: adjust_creative
+  type: STRING
+  mode: NULLABLE
+  description: Adjust Creative the client is attributed to.
+
+- name: adjust_network
+  type: STRING
+  mode: NULLABLE
+  description: Adjust Network the client is attributed to.
+
+- name: ping_sent_metric_date
+  type: INTEGER
+  mode: NULLABLE
+  description: Number of clients that sent a ping on the metric date.
+
+- name: ping_sent_week_4
+  type: INTEGER
+  mode: NULLABLE
+  description: Number of clients that sent a ping 4 weeks after the metric date.
```

Comment:
both on the metric date and 4 weeks after the metric date

Reply:
actually let's call it "4th week after" instead of "4 weeks after" since it's between days 22-28